### DISCUSSION TITLE
hide the language switcher icon from the Windows 11 style taskbar

### DISCUSSION THREAD

**Discussion by tp01-gogogo** [valinet/ExplorerPatcher#3245] [Ideas]
remove the language switcher icon from the Windows 11 style taskbar

**Comment by tp01-gogogo** [comment]
feature request: keyboard switcher/language switcher button will be always hidden whatever i switch language or restart PC when i use win 11 style taskbar.

**Comment by tp01-gogogo** [comment]
We can hide language switcher icon on Win 10, but on Win11 style taskbar, there is no such setting.

**Comment by pyrates999** [comment]
https://www.minitool.com/news/turn-on-off-the-language-bar.html

**Reply by tp01-gogogo** [reply]
It does't work on Win11.

**Reply by pyrates999** [reply]
does it work if you uninstall EP?

**Reply by tp01-gogogo** [reply]
> does it work if you uninstall EP?

No...it doesn't work on win11, so i hope EP win11 style will have the funciton. On win11, i followed steps on https://www.minitool.com/news/turn-on-off-the-language-bar.html , it will come back to default if i lock screen or restart Windows.

**Reply by pyrates999** [reply]
you mean if you turn off that setting, the language bar will not be shown.  But if you restart your computer or lock the screen, the language bar will be shown again?

**Reply by tp01-gogogo** [reply]
> you mean if you turn off that setting, the language bar will not be shown. But if you restart your computer or lock the screen, the language bar will be shown again?

On win11, if I turn on that setting, the language bar will not be shown. But if I restart my computer or lock the screen or switch language, the language bar will be shown again. I hope language bar does not show on system tray with win11 style EP.
![turn-on-off-the-language-bar-3](https://github.com/user-attachments/assets/0fc83d46-f99b-492b-bac7-1a176e8c8438)

**Reply by pyrates999** [reply]
you mean if you untick **Use the desktop language bar when it's available**, it will not be shown until you restart your computer or you lock your computer?

**Reply by tp01-gogogo** [reply]
> you mean if you untick **Use the desktop language bar when it's available**, it will not be shown until you restart your computer or you lock your computer?
I tick **Use the desktop language bar when it's available** , but if I restart my computer or lock my computer, sometimes it may be back to untick, I tested on all my win11 PC, so I hope EP will have the function that hide language bar with win11 style.

**Reply by pyrates999** [reply]
since you're using the windows 11 style taskbar, can you uninstall EP and see if this is still happening?  The settings option unticking itself would not be something that EP would cause I don't think.  But lets check it anyways.

**Reply by tp01-gogogo** [reply]
> since you're using the windows 11 style taskbar, can you uninstall EP and see if this is still happening? The settings option unticking itself would not be something that EP would cause I don't think. But lets check it anyways.

I checked, it's not EP's problem, but could EP support hide language bar while using the win11 style taskbar? Needs that function.

**Reply by pyrates999** [reply]
That would be for MS to fix, not EP.  Sorry.  Please close this.

**Comment by bernhardbarker** [comment]
For me, the icon pops back up again every other time there's a Windows update. In that case, "Use the desktop language bar when it is available" remains checked, and unchecking and rechecking it fixes the problem.

It doesn't come back for me when I restart or lock, so you might be having more issues.

It would be good if ExplorerPatcher could implement this feature, to remove it permanently and properly.
